File: services/order-service/app/src/main/java/io/micro_spring_boot/order_service/app/OrderServiceApplication.java
package io.micro_spring_boot.order_service.app;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(
    scanBasePackages = {
        "io.micro_spring_boot.order_service.app",
        "io.micro_spring_boot.order_service.infra.configurations",
    }
)
public class OrderServiceApplication {
    public static void main(String[] args) {
        SpringApplication.run(OrderServiceApplication.class, args);
    }
}

File: services/order-service/app/src/main/kotlin/com/microspringboot/orderservice/app/OrderServiceApplication.kt
package com.microspringboot.orderservice.app

import jakarta.annotation.PostConstruct
import org.springframework.boot.Banner
import org.springframework.boot.WebApplicationType
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.domain.EntityScan
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing
import org.springframework.data.jpa.repository.config.EnableJpaRepositories
import java.time.ZoneId
import java.util.*
import java.util.TimeZone.getTimeZone

@SpringBootApplication(scanBasePackages = ["com.microspringboot.order_service"])
@EnableJpaAuditing
@EnableJpaRepositories(value = ["com.microspringboot.order_service.infra.repositories"])
@EntityScan(value = ["com.microspringboot.order_service.domain.aggregates"])
class OrderServiceApplication {
    @PostConstruct
    fun started() {
        TimeZone.setDefault(getTimeZone(ZoneId.of("UTC")))
    }
}

fun main(args: Array<String>) {
    runApplication<OrderServiceApplication>(*args) {
        setBannerMode(Banner.Mode.OFF)
        webApplicationType = WebApplicationType.REACTIVE
    }
}
